### ISSUE TITLE
tableswitch block depth?

### ISSUE THREAD

**Issue by JSStats** [WebAssembly/sexpr-wasm-prototype#32]
The following code fails to encode with sexp-wasm with: t60.wast:7:37: label variable out of range (max 1)

If an explicit label is added to the `tableswitch` it succeeds. I've seen other cases where this changes the depths. Does v8 always have an implicit block for a `tableswitch`? If so then adding an explicit label should make no difference, and if not then how is the difference encoded?

The v8 encoding seems to accept just one expression per case, so emits a block if there are more than one statement. I assume there is some logic to offset the `br` depths and might it only be in this case that the depths would be modified?

```
(module
  (func $f1 (param $i i32) (result i32)
    (block $l1
      (i32.add (tableswitch (get_local $i)
                 (table (case $0)) (case $default)
                 (case $0 (br 0 (i32.const 1)))
                 (case $default (br 1 (i32.const 2))))
               (i32.const 3))))
  (export "f1" $f1))
(invoke "f1" (i32.const 0))
(invoke "f1" (i32.const 1))

4 : i32
2 : i32
```

**Comment by binji** [comment]
It used to be that break depths in the spec interpreter depended on whether a label was specified. The v8 binary format always introduces a new break depth for `block` and `tableswitch` and adds two for `loop`. So there was a discrepancy in the break depths, but it looks like there shouldn't be now.

**Comment by binji** [comment]
Fixed in aa3940ba53d039aed322abb2e7aba3716f755890.
